### ISSUE TITLE
Publishing to Supermarket?

### ISSUE THREAD

**Issue by ColmHally** [PaytmLabs/chef-ixgbevf#1]
Hi there,
Just wondering if you plan on publishing this cookbook to the Chef Supermarket?
Regards,
Colm

**Comment by jhohertz** [comment]
Yes, I do, there's a couple of little things I want to fix up first before I do. The main one being that there are some poor interactions with other cookbooks which rely on build-essential, which can cause the kernel headers package installed to become mis-aligned with the currently running kernel.

The workaround for now has been to ensure I am only running it on systems that are already up-to-date.

The real fix is to let this cookbook run in the compile phase so it can "beat" other cookbooks bringing in build-essential, or for me to have it do something more elaborate for detecting the most recently installed kernel, and apply the appropriate flags to DKMS to compile to that kernel vs. the current running one. I don't actually like either approach, open to others...

So I am holding off publishing it wider until I have time to address this particular issue.

**Comment by ColmHally** [comment]
Thanks for the reply. That makes sense; I ran into this issue myself and fixed it by ensuring the correct headers were installed before including your recipe.
Not the nicest solution, but I'm stuck for time!

```
package "linux-headers-#{node['os_version']}"
include_recipe 'ixgbevf::default'
```
